src/components/media-player/html5-player.less

```diff
@@ -19,6 +19,7 @@
 
 .Html5Player-playOverlay {
   .m-video-play-overlay;
+  opacity: 0;
```

Comment:
There will be a discrepancy with the `youtube-player` now, but since there are but 2 usages of this mixin, the opacity could go into the mixin.

Reply:
Removed, see the next comment.